site/content/docs/5.0/components/offcanvas.md

```diff
@@ -119,14 +119,14 @@ Try the right and bottom examples out below.
 
 ## Options
 
-By default, we disable scrolling on the `<body>` when an offcanvas is visible and use a gray backdrop. Use the `data-bs-body` attribute to enable `<body>` scrolling, or a combination of both options
+By default, we disable scrolling on the `<body>` when an offcanvas is visible and use a gray backdrop. Use the `data-bs-scroll` attribute to enable `<body>` scrolling, and `data-bs-backdrop` attribute to achieve a combination of both options
```

Comment:
User can not achieve the combination of both options using `data-bs-backdrop="true"`